cmd2/rl_utils.py

```diff
@@ -141,7 +141,7 @@ def pyreadline_remove_history_item(pos: int) -> None:
 
 elif 'gnureadline' in sys.modules or 'readline' in sys.modules:
     # We don't support libedit. See top of this file for why.
-    if 'libedit' not in readline.__doc__:
+    if readline.__doc__ and 'libedit' not in readline.__doc__:
```

Comment:
I would prefer this to be `readline.__doc__ is not None` just to be clearer about what you're checking.

Reply:
done